internal/config/apparmor/apparmor.go

```diff
@@ -38,9 +37,9 @@ func (c *Config) LoadProfile(profile string) error {
 		return nil
 	}
 
-	if profile == unconfined {
+	if profile == v1.AppArmorBetaProfileNameUnconfined {
 		logrus.Info("AppArmor profile is unconfined which basically disables it")
-		c.defaultProfile = unconfined
+		c.defaultProfile = v1.AppArmorBetaProfileNameUnconfined
```

Comment:
This should be a separate commit, please.

Reply:
Will be a dedicated PR, thank you for the hint.

Reply:
See https://github.com/cri-o/cri-o/pull/4098